### PR TITLE
[APPTS-10016] Address dependency vulns in appc-platform-sdk

### PR DESCRIPTION
**Pull request by feons**
https://techweb.axway.com/jira/browse/APPTS-10016

1. replace `nps check` with `npm audit` since the service is not longer working.
https://blog.npmjs.org/post/175511531085/the-node-security-platform-service-is-shutting
2. include the change proposed by `npm audit fix`

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 library 'pipeline-library'
 
 def nodeVersion = '6.11.3'
```
Comment (pkclark) [inline Jenkinsfile]: @feons wanna bump this too?  LTS only going forward.

**Comment by feons** [comment]
@pkclark, dep bumped.

**Comment by feons** [comment]
@pkclark, need to check `developer_org_id` is a valid org for testing.

**Comment by pkclark** [comment]
appc-platform-sdk@3.1.1 published. Thank you @feons

**Pull request merged**